ExampleMod/ExamplePlayer.cs

```diff
@@ -634,6 +635,103 @@ public class ExamplePlayer : ModPlayer
 			}
 		}
 
+		// This is an example of a glowmask on an equipable item. Realized via PlayerLayer
+		// You'll find the associated armor for this by searching for "ExampleBreastplate" in ExampleMod
+
+		// We start off by declaring a PlayerLayer, and give it our mods name, the layers name, its parent layer, and then a method that contains our drawing code
+		// This PlayerLayer we create will be added to the list of visible PlayerLayers in the ModifyDrawLayers hook below
+		public static readonly PlayerLayer ExampleBreastplateGlowmask = new PlayerLayer("ExampleMod", "ExampleBreastplateGlowmask", PlayerLayer.Body, delegate (PlayerDrawInfo drawInfo) {
+			
+			// We don't want the glowmask to draw if the player is cloaked or dead
+			if (drawInfo.shadow != 0f || drawInfo.drawPlayer.dead) {
+				return;
+			}
+
+			Player drawPlayer = drawInfo.drawPlayer;
+			Mod mod = ModLoader.GetMod("ExampleMod");
+
+
+			// To figure out when we should draw our ExampleBreastplate glowmask, we make use of GetEquipSlot.
+			// It takes in the name of the item that comes with an equip texture, and then its EquipType.
+			// By comparing that to drawPlayer.body, which represents the currently visible body armor, we make sure it's safe to draw ours
+			// Here we abort if the currently equipped body armor isn't what we need for our glowmask
+			if (drawPlayer.body != mod.GetEquipSlot("ExampleBreastplate", EquipType.Body)) {
+				return;
+			}
+
+			// Because we have a breastplate glowmask, and breastplates have different textures depending on if the player is male or female, we have to use two textures here
+			string gender = "";
```

Comment:
Personally prefer:
`Texture2D texture = drawPlayer.Male ? mod.GetTexture("Items/Armor/ExampleBreastplate_Body_Glowmask") : mod.GetTexture("Items/Armor/ExampleBreastplate_Female_Body_Glowmask");`
You don't use gender elsewhere, so with this it's easier and faster to see the effect of the gender